CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## 0.1.1
+
+- Add missing semantic colors to harmonization from updated m3 color scheme
```

Comment:
Is there some way to mention that we require the master channel to use this new version?

Reply:
Yes, see the flutter SDK constraint in the pubspec